etl/assets.py

```diff
@@ -83,43 +78,37 @@ def documents_of_wikipedia_articles_with_summaries(
 @asset
 def wikipedia_articles_embedding_store(
     documents_of_wikipedia_articles_with_summaries: DocumentTuple,
-    openai_settings: OpenaiSettings,
-    input_config: InputConfig,
+    pipeline_config: PipelineConfig,
```

Comment:
Discussed removing enrichment type from this.

Reply:
I have removed `EnrichmentType` from the project.